omod-common/src/test/java/org/mockingbird/test/rest/resource/AnimalClassResource_1_9.java

```diff
@@ -9,6 +9,7 @@
  */
```

Comment:
They have always been there. I only implemented the missing methods. It seems that they are only used by `RestServiceImplTest`